<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>base</class>
 <widget class="QWidget" name="base">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QStackedWidget" name="top_display">
     <widget class="QWidget" name="keyframe_configuration">
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>340</x>
         <y>300</y>
         <width>161</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>Keyframe Configuration</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="timeline_configuration">
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>350</x>
         <y>290</y>
         <width>211</width>
         <height>91</height>
        </rect>
       </property>
       <property name="text">
        <string>Timeline Configuration</string>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="timeline">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>200</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::Shape::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Shadow::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2"/>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
